docker-compose.https.yml

```diff
@@ -3,6 +3,9 @@ services:
   jobservice:
     environment:
       SSL_TOMCAT_CA_CERT_LOCATION: /keystore/tomcat.keystore
+      SSL_TOMCAT_CA_CERT_KEYSTORE_PASS: ${JOB_SERVICE_KEYSTORE_PASS:-changeit}
+      SSL_TOMCAT_CA_CERT_KEY_PASS: ${JOB_SERVICE_KEY_PASS:-changeit}
```

Comment:
I'm going to be pedantic here but I don't like these environment variables ending in `_PASS` - it should be `_PASSWORD`.